packaging/homebrew/README.md

```diff
@@ -0,0 +1,34 @@
+# Building a Homebrewbrew package of the CLI
+
+The Homebrew package of the CLI is an empty metapackage that installs the CLI gem directly through Ruby. Therefore, the
+package itself only serves to facilitate the installation and inclusion of non-ruby dependencies.
```

Comment:
"non-Ruby"?

Reply:
That makes sense, but thanks for bringing this up, this one slipped past me. This is NOT a meta-package, but I'll make sure to fix the docs for the ones that are.